### ISSUE TITLE
Admin UI touching not working in pages

### ISSUE THREAD

**Issue by testt23** [luyadev/luya#1948]
Hi everybody,
An important problem that hinders the system when using a cellphone.  When you add a new page you can't add items with the touchpad on her phone, that means you can't post a new page successfully, the problem is terrible.

**Comment by nadar** [comment]
Yes we are aware of this, but we also have to admit that the CMS page management is not made for mobile browser, its made for desktops. If there is an easy way to improve, we could do that but for now i have no idea how (in a fast and easy way!).

As CRUD data management is more a common mobile task, the CRUD is working nice on such devices. Maybe @TheMaaarc has something to mention regarding responsive a cms block editor.
